securedrop/source_templates/index.html

```diff
@@ -88,7 +88,7 @@ <h2 class="welcome-text">
     <div class="bubble" id="security-slider-info-bubble">
       <p>{{ gettext('You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!') }}</p>
       <ol>
-        <li>{{ gettext('Click the') }} <img src="{{ url_for("static", filename="i/toronion.png") }}" alt="Tor icon">{{ gettext('Tor icon in the toolbar above') }}</li>
+        <li>{{ gettext('Click the <img src="{icon}" alt="Tor icon"> Tor icon in the toolbar above').format(icon=url_for("static", filename="i/toronion.png"))  }}</li>
         <li>{{ gettext('Click <strong>Security Settings...</strong>') }}</li>
         <li>{{ gettext('Turn the Slider to <strong>High</strong>, then click <strong>Ok</strong>') }}</li>
         <li>{{ gettext('<a href="/">Click here</a>to refresh the page') }}</li>
```

Comment:
A space is needed before **to refresh**. You did not do that but I noticed it now ;-)

Reply:
Fixed